### PR TITLE
docs: Stylelint at-rule-empty-line-before

### PR DESCRIPTION
**Pull request by nschonni**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
Not sure if this is required, currently I see the space is to group imports like `tokens`, `components`, etc.

**Comment by nschonni** [comment]
> Not sure if this is required, currently I see the space is to group imports like `tokens`, `components`, etc.

Yeah, don't feel strongly about this one at all. I think it is hinting that those blank lines might be missing a comment to indicate what the logical grouping is about

File: docs/src/assets/scss/styles.scss [hunk 1]
```diff
@@ -13,7 +12,6 @@
 @import "docs";
 @import "versions";
 @import "languages";
-
 @import "components/buttons";
 @import "components/docs-navigation";
 @import "components/toc";
```
Comment (mdjermanovic) [inline docs/src/assets/scss/styles.scss:17]: I agree with @snitin315. These empty lines are used to group imports, and adding comments seems unnecessary here, so perhaps we shouldn't enable this rule.

**Comment by snitin315** [comment]
Thanks for the PR but we have decided to not enable this rule.

**Pull request closed without merging**